client-mvp-04/package.json

```diff
@@ -55,18 +57,19 @@
     ]
   },
   "devDependencies": {
-    "eslint-config-airbnb": "^18.2.0",
-    "eslint-config-airbnb-base": "^14.2.0",
-    "eslint-config-prettier": "^6.11.0",
-    "eslint-plugin-import": "^2.22.0",
-    "eslint-plugin-jest-dom": "^3.6.5",
-    "eslint-plugin-jsx-a11y": "^6.3.1",
-    "eslint-plugin-prettier": "^3.1.4",
-    "eslint-plugin-react": "^7.20.6",
-    "eslint-plugin-react-hooks": "^4.1.2",
-    "eslint-plugin-testing-library": "^3.10.1",
+    "eslint": "^6.6.0",
```

Comment:
This version is outdated. Please use the freshest version from https://eslint.org/blog/

Reply:
Another one thing regarding versions. I got some warnings related to ESLint versions when running `yarn install` for `client-mvp-04`:
```
warning " > eslint-config-prettier@8.3.0" has incorrect peer dependency "eslint@>=7.0.0".
warning "eslint-plugin-jest > @typescript-eslint/experimental-utils > @typescript-eslint/typescript-estree > tsutils@3.21.0" has unmet peer dependency "typescript@>=2.8.0 || >= 3.2.0-dev || >= 3.3.0-dev || >= 3.4.0-dev || >= 3.5.0-dev || >= 3.6.0-dev || >= 3.6.0-beta || >= 3.7.0-dev || >= 3.7.0-beta".
warning " > eslint-plugin-testing-library@4.9.1" has incorrect peer dependency "eslint@^7.5.0".
```
Please pay attention to such messages when you update the ESLint version. It's a fresh addition to the project packages so it will be nice to have no warnings at least at the beginning.

Reply:
Actually, I have used the latest version first. Then I was getting client-mvp-04 is not compatible with that version use `^6.6.0` like this kind of things. Maybe it's because creating an app with `create-react-app`  comes with `eslint` by default.  I will try to solve this.

Reply:
![image](https://user-images.githubusercontent.com/43095489/126267153-4263d2fd-55c5-408f-8000-9aafd8899197.png)
I think it's better not to add manually. I had added that by thinking for some developers' eslint package is not installing having some problem with installation package by the `react-script`.

Reply:
What do you think about enabling  `SKIP_PREFLIGHT_CHECK=true` ?

Reply:
What do you think about enabling  `SKIP_PREFLIGHT_CHECK=true` ?
Adding `SKIP_PREFLIGHT_CHECK=true` on environment variable and upgrading to the latest `eslint` version removes all warnings like `unmet peer dependency`  and `incorrect peer dependency`